src/js/utils/graphics.js

```diff
@@ -13,7 +13,7 @@ export const polarToCartesian = (centerX, centerY, radius, angleInDegrees) => {
 export const arcCommands = (centerX, centerY, radius, startAngle, endAngle) => {
   // handle that we can't draw a complete circle
   let normalizedEndAngle = endAngle;
-  if (endAngle - startAngle >= 360) {
+  if (endAngle > startAngle && endAngle - startAngle >= 360) {
```

Comment:
Can we add a comment around what this logic does? especially since now we're updating it